### DISCUSSION TITLE
HASHICROP VAULT CONNECTION

### DISCUSSION THREAD

**Discussion by Akashs12345** [apache/airflow#37248] [Q&A]
### Description

we trying to connect airflow(2.7.0) to the hashicrop vault (1.14.0) we are using a helm chart for both deployments Please guide us on how to configure the vault in airflow 

thank you in advance

### Use case/motivation

connection between the vault and airflow 

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Please don't use Github Issues for such questions - use discussions instead.

**Reply by Akashs12345** [reply]
ok sorry my bad can guide me how can i configure the airflow and vault connection in helm charts


thank you

**Reply by Akashs12345** [reply]
waiting for a suggestion anyone?

**Reply by potiuk** [reply]
Maybe time to attempt it yourself - searching the docs. You are not guaranteed to get answer here - people here might help if they decide to spend their free time away of their familiies and day jobs, to help if they have experience and see they can help.

My suggestion is - if you want to tap into such free help - is to show what you've done, what you already tried following the documentation, where you failed, what problems you faced where you got stuck. That usually help people to decide if they see that you've 'done your part' and look for help or if you just expect to get free help without any investment from your side.

That would be my advice.

**Reply by Taragolis** [reply]
Just keep it there, because it is not clear what actual problem here
- [Vault Connection](https://airflow.apache.org/docs/apache-airflow-providers-hashicorp/stable/connections/vault.html)
- [Hashicorp Vault Secrets Backend](https://airflow.apache.org/docs/apache-airflow-providers-hashicorp/stable/secrets-backends/hashicorp-vault.html)
- [Managing Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
- [Secrets Backend](https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/index.html)
- [Helm Chart for Apache Airflow: Adding Connections, Variables and Environment Variables](https://airflow.apache.org/docs/helm-chart/stable/adding-connections-and-variables.html)
